contract-message-predicate/src/transaction_utils.sw

```diff
@@ -0,0 +1,117 @@
+library transaction_utils;
+
+use std::mem::read;
+use std::constants::ZERO_B256;
+
+const GTF_SCRIPT_GAS_PRICE = 0x002;
+const GTF_SCRIPT_GAS_LIMIT = 0x003;
+const GTF_SCRIPT_SCRIPT_LENGTH = 0x005;
+const GTF_SCRIPT_SCRIPT = 0x00B;
+const GTF_SCRIPT_INPUTS_COUNT = 0x007;
+const GTF_SCRIPT_OUTPUTS_COUNT = 0x008;
+
+const GTF_INPUT_TYPE = 0x101;
+const GTF_INPUT_COIN_AMOUNT = 0x105;
+const GTF_INPUT_COIN_ASSET_ID = 0x106;
+const GTF_INPUT_CONTRACT_CONTRACT_ID = 0x113;
+const GTF_INPUT_MESSAGE_DATA_LENGTH = 0x11B;
+const GTF_INPUT_MESSAGE_DATA = 0x11E;
+
+const GTF_OUTPUT_TYPE = 0x201;
+const GTF_OUTPUT_CONTRACT_INPUT_INDEX = 0x205;
```

Comment:
Can you put a TODO/good first issue to replace these with constants from the stdlib?